roles/install-devstack/tasks/main.yml

```diff
@@ -45,6 +45,9 @@
       export PROJECTS="openstack/devstack-plugin-container $PROJECTS"
       export PROJECTS="openstack/kuryr-libnetwork $PROJECTS"
       export PROJECTS="openstack/zun openstack/zun-tempest-plugin $PROJECTS"
+      export PROJECTS="openstack/diskimage-builder $PROJECTS"
+      export PROJECTS="openstack/tripleo-image-elements $PROJECTS"
+      export PROJECTS="openstack/octavia openstack/python-octaviaclient $PROJECTS"
```

Comment:
tripleo-image-element and octavia have been in PROJECTS, we only need add diskimage-builder and python-octaviaclient

https://github.com/openstack-infra/devstack-gate/blob/master/devstack-vm-gate-wrap.sh#L105-L119